### ISSUE TITLE
epic-games captch

### ISSUE THREAD

**Issue by Dxrmy** [vogler/free-games-claimer#288]
The epic-games captcha is back.
Even when running in both modes, and is now unsolvable where before it wasn't.

**Comment by EQUALIT-CG** [comment]
Hello Dxrmy

I 2nd this I just cleaned up a lot of my docker and can't sing into epic 
![Capture](https://github.com/vogler/free-games-claimer/assets/56815873/fb3325fa-ed92-42f1-a467-1947cec1f150)


Iv removed my email from the screenshot Iv not tried any other log in as of yet because i want to get one done at a time until then...

please lets get this fix @vogler

**Comment by Comamndershephard** [comment]
yeah been an issue for a month now for me, just can't sign in, captcha pops up, just says its wrong once completed

**Comment by blakeseufert** [comment]
Having the same issue for quite some time. Apparently this is a common bug on safari too.

**Comment by greenmojo2** [comment]
bump

**Comment by BGarber42** [comment]
Duplicate of #183

**Comment by Luckyfive** [comment]
having the same issue here, first time user so I wasn't sure if it was out of the norm

**Comment by dmi97** [comment]
Epic games captcha is still showing up despite getting a new IP address, and it is unsolvable in the browser.

```
  Got hcaptcha challenge! Lost trust due to too many login attempts? You can solve the captcha in the browser or get a new IP address.
locator.waitFor: Timeout 60000ms exceeded.
=========================== logs ===========================
waiting for locator('text=Thanks for your order!')
============================================================
    at /fgc/epic-games.js:237:59 {
  name: 'TimeoutError'
}
  Failed to claim! To avoid captchas try to get a new IP address.

```

**Comment by EQUALIT-CG** [comment]
@vogler hello

I'm not trying to hassle you but may i ask if that fine
Not why you closed it that up to you but is there any idea if you're able to fix it or how myself and many other can fix it please so we can use this to claim out games... if you don't want to that fine but I'm just carious as to how to fix it.

if its not maybe on your main page put EOL "end of life support" or something i would love to have this running again but I'm lost as to what to do

anyways its really good to see you again. i know its hard but well done coming back.

**Comment by vogler** [comment]
Closed it as duplicate of #183 as mentionend above.
As I've already written there, for me everything works fine outside of docker. Sometimes I also get a captcha when I'm testing too much, but otherwise no issues.
You can read the comments there and try out different fingerprinting approaches if you want a fix for docker.
I'm not keen on investigating it since it's just trial and error and unclear how long it would take.

Some people apparently also get a captcha outside of docker. For those, you'd have to collect all data about their environments to find out what the issue may be - lots of work, outcome also unclear.

**Comment by EQUALIT-CG** [comment]
Hi @vogler 

no worries that fine

I was just carious as i had it running but i did read that and also hard reset my public ip etc etc but i thought it maybe the bower out of data that all was waiting for an update or something so hum... etc etc

like when chrome or Firefox get update or out of date they no longer work like internet exp etc etc.

but that fine i will take it on the chin as i have been trying many times but not working thank you for your reply.
